imagetest/test_suites/image_validation/license_test.go

```diff
@@ -116,9 +116,10 @@ var licenses = []string{
 	`Netcat and the associated package is a product of Avian Research, and is freely available in full source form with no restrictions save an obligation to give credit where due.`,
 	`Permission is hereby granted, without written agreement and without licence or royalty fees, to use, copy, modify, and distribute this software`,
 	`Open Market permits you to use, copy, modify, distribute, and license this Software and the Documentation for any purpose, provided that existing copyright notices are retained in all copies and that this notice is included verbatim in any distributions. No written agreement, license, or royalty fee is required for any of the authorized uses.`,
-	`This software is made available under the terms of *either* of the licenses found in LICENSE.APACHE or LICENSE.BSD. Contributions to cryptography are made under the terms of *both* these licenses.`,
+	`This software is made available under the terms of *either* of the licenses found in LICENSE.APACHE or LICENSE.BSD.`,
 	`This library.*is public domain software`,
-	`This Font Software is licensed under the SIL Open Font License`,
+	`Apache License`,
```

Comment:
this does not belong here. we first try for license NAME, then we try license TEXT REGEX. this is the latter, but the string you've added belongs in the former.

Reply:
The issue is with the regex... because its looking for a other context before the license name. For whatever reason, some of the apache licensed packages don't have that... the simply start with Apache License ... so the regex fails. I don't know the history of this regex however.

Reply:
same for 'the artistic license 2.0'

Reply:
Like I said, this doesn't work. Do you know why we have the wrapper text in the regex?
`licenseNameRegex = `(?i)(((License|Copyright)\s*:\s*%[1]s)|((covered )?under (the )?%[1]s)|(under (the terms of )?the %[1]s))``

The issue with several packages is that they don't conform to this.

Reply:
my comments were supposed to be back to back :)
thanks for explaining, i think this means we should approach the design again from scratch, but fine to proceed like this for now due to the limitations.